app/models/conference.rb

```diff
@@ -6,6 +6,8 @@ class Conference < ActiveRecord::Base
   resourcify :roles, dependent: :delete_all
 
   default_scope { order('start_date DESC') }
+  scope :upcoming_conferences, (-> { where('end_date >= ?', Date.current) })
+  scope :past_conferences, (-> { where('end_date < ?', Date.current) })
```

Comment:
looks like it has been fixed here https://github.com/bbatsov/rubocop/pull/4237 . but the fix is not backported to the version we are using i.e. `0.48.1` .